tests/utils/mqtt_broker.py

```
@@ -0,0 +1,194 @@
+    def running_processes(cls) -> list[subprocess.Popen]:
+        return cls._running_broker_processes
+
+    @classmethod
+    def kill_all_test_brokers(cls):  # pragma: no cover
+        for process in cls._running_broker_processes:
+            print(f"Killing test broker process '{process.pid}'.")
+            process.kill()
+            try:
+                cls._running_broker_processes.remove(process)
+            except ValueError:
+                pass
```

Ruff

Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ValueError)`

(SIM105)